documentation/synthetic_tokens/tutorials/liquidation.md

```diff
@@ -0,0 +1,423 @@
+
+To facilitate this configuration the UMA liquidator bots have an optional configuration object that can be used to specify a `crThreshold` which defines how far below the given CR ratio a position must fall before the bot will initiate the liquidation. To enable this configuration with a 5% threshold add the following to your `liquidator.env`:
+
+```
+LIQUIDATOR_CONFIG={"crThreshold":0.05}
+```
+
+This configuration has additional optional configurations including:
+
+1. `liquidationDeadline` which aborts the liquidation if the transaction is mined this amount of time after the EMP client's last update time. Defaults to 5 minutes.
+2. `liquidationMinPrice` which aborts the liquidation if the amount of collateral in the position per token outstanding is below this ratio. Defaults to 0.
+
+These configurations can be added to the config in the same way the `crThreshold` was added.
+
+## Appendix: Bot configuration parameters
+
+This tutorial touched on the key configuration parameters available when running an UMA liquidation or dispute bot.
+There are a few more configuration options available. The section below describes the parameter input in this tutorial as well as the optional extra parameters that can be included when running a bot.
+
+- `POLLING_DELAY`**[required]**: how long the bot should wait (in milliseconds) before running a polling cycle.
+- `EMP_ADDRESS`**[required]**: address of the deployed expiring multi party contract on the given network you are wanting to connect to. This config defines the synthetic that the bot will be liquidating.
+- `MNEMONIC`**[required]**: generated before hand or in the steps outlined in key generation.
+- `PRICE_FEED_CONFIG`**[required]**: configuration object used to parameterize the bot's price feed. It's broken down as follows:
+  - `type` spesifies the configuration of the price feed. The `medianizer` type averages the price of the identifier over a set of diffrent exchanges.
+  - `apiKey` is the key generated in API key section of the Prerequisites.
+  - `pair` defines the crypto pair to medianizer over.
+  - `lookback` defines a window size, in seconds, over which an average price is taken.
+  - `minTimeBetweenUpdates` min number of seconds between update.
+  - `medianizedFeeds` is an array of type `priceFeed` that defines the feeds overwhich the meadinzer will average. Each of these have their own components which are defined as:
+    - `type` Each instance of the meadinaizer also a type. This could be a `medianizer`, `uniswap` or `cryptowatch` depending on the configuration of the bot. The sample bot is using only `cryptowatch` price feeds to average over the set of exchanges to medianize.
```

Comment:
```suggestion
    - `type` Each instance of the meadinaizer also a type. This could be a `medianizer`, `uniswap` or `cryptowatch` depending on the configuration of the bot. The sample bot is using only `cryptowatch` price feeds to compute the median.
```